experimenter/experimenter/experiments/api/v5/serializers.py

```diff
@@ -833,9 +833,11 @@ class NimbusExperimentSerializer(
         allow_null=True,
         required=False,
     )
-    proposed_release_date = serializers.DateField(
-        allow_null=True,
+    proposed_release_date = serializers.CharField(
+        min_length=0,
+        max_length=1024,
         required=False,
+        allow_blank=True,
```

Comment:
Oh what I was saying the other day was setting it to a string in the GQL but keeping it as a DateField in the serializer so it converts back to a date.